packages/typescript/src/plugins/FederatedTypesPlugin.ts

```diff
@@ -137,7 +137,8 @@ export class FederatedTypesPlugin {
       try {
         this.logger.log(`Getting types index for remote '${remote}'`);
         const resp = await axios.get<TypesStatsJson>(
-          `${origin}/${this.normalizeOptions.typesIndexJsonFileName}`
+          `${origin}/${this.normalizeOptions.typesIndexJsonFileName}`,
+          { timeout: 1000 }
```

Comment:
id have a longer timeout or allow an option for userland to override and set timeout, 1 second might be cause limits on slow connections or under pressure boxes. Id set timeout to like 2000 by default

Reply:
done